src/mixins/input.js

```diff
@@ -68,8 +69,13 @@ export default {
   mounted () {
     this.$nextTick(() => {
       const input = this.$refs.input
-      if (this.autofocus && input) {
-        input.focus()
+      if (input) {
+        if (this.autoselect) {
+          input.select()
+        }
+        if (this.autofocus) {
```

Comment:
this should be an else if

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/select
> Calling element.select() will not necessarily focus the input, so it is often used with HTMLElement.focus().

I think there is right logic, if we want to use both "autofocus" and "autoselect"...

Reply:
In this case it should be:
```javascript
if (this.autoselect) {
  input.select()
  input.autofocus()
}
else if (this.autofocus) {
  input.autofocus()
}
```

or

```javascript
if (this.autoselect) {
  input.select()
}
if (this.autoselect || this.autofocus) {
  input.autofocus()
}
```

as it makes no sense to select but not to focus.

Reply:
>In this case it should be..

Yes, it makes sense.

>input.autofocus()

Probably you meant input.focus() ? 

Also can change type of props "autofocus" to [String, Boolean], and check if it is equal to "select" or true...

Reply:
sorry, .focus ()

Reply:
So i suggest **Line 3** replace to:
```
autofocus: [String, Boolean],
```

And after **Line 72** add:
```
if (this.autofocus === 'select') {
  input.select()
}
```

Reply:
I realized that not every time you need `autoselect`, you also need `autofocus`. So that `else if` block won't work well.
Consider the case when you are on a mobile and you don't want the keyboard to popup because you don't need the user to input anything, but press a "Copy this" button next to the input with its text selected. Or in desktop you simply let the user press `ctrl + c`.
They are 2 different functionalities, that's why we can't use ´else if´ here.
Regarding to `this.autofocus === 'select'`, I agree